docs/rendering-tip.md

```diff
@@ -1,2 +1,50 @@
 # Rendering Tips
+Hubble offers the following options to render your visualizations:
+- GIF
+- MP4
```

Comment:
I don’t think we can say MP4 is supported. Instead I think you could create a section for converting webm or a photo sequence into an mp4 using ffmpeg and/or one of those conversion websites.

Reply:
I suppose that's true for now. I put it in there for the case that I figured out how to make an MP4 that would natively play on Mac. Right now I know we could easily make another encoder for MP4 by switching L79 https://github.com/uber/hubble.gl/blob/6c044c423c43d37c62fd396fdd6f2a96c885a66a/modules/core/src/encoders/video/stream-encoder.js#L79 . Should I just remove it for now since this will be pushed to production on the Github pages?

Reply:
Yes, I think we should only publish what we currently support. 

I don’t think the StreamEncoder change is ideal, since it drops frames and has playback issues. It’s probably not a good idea to put it as the flagship MP4 encoder.

Reply:
Noted. Will remove it for now